api/pkg/backend/service.go

```diff
@@ -347,17 +347,24 @@ func (s *APIService) DeleteBackend(request *restful.Request, response *restful.R
 
 	ctx := common.InitCtxWithAuthInfo(request)
 	// TODO: refactor this part
```

Comment:
IS the TODO still needed ? Please remove , if already fixed.

Reply:
removed